butterfree/clients/cassandra_client.py

```diff
@@ -16,6 +16,12 @@
 from typing_extensions import TypedDict
 
 from butterfree.clients import AbstractClient
+from butterfree.configs.logger import __logger
+
+logger = __logger("cassandra_client")
+
+EMPTY_STRING_HOST_ERROR = "The value of Cassandra host is empty. Please fill correctly with your endpoints"  # noqa: E501
```

Comment:
If we're importing this strings elsewhere in our code, isn't it better to store them in a `cassandra_client_errors.py` file, in an Enum, or something like that?

Reply:
I don't see this need. It's just used there. If something changes we can create something more generic

Reply:
Oh, I see! Got confused thinking it was used somewhere else :)